client/cloud_credentials.go

```diff
@@ -75,6 +79,22 @@ type GcpCredentialsValuePayload struct {
 	ServiceAccountKey string `json:"serviceAccountKey"`
 }
 
+func (c *GoogleCostCredentialsCreatePayload) SetOrganizationId(organizationId string) {
+	c.OrganizationId = organizationId
+}
+
+func (c *AwsCredentialsCreatePayload) SetOrganizationId(organizationId string) {
+	c.OrganizationId = organizationId
+}
+
+func (c *GcpCredentialsCreatePayload) SetOrganizationId(organizationId string) {
+	c.OrganizationId = organizationId
+}
+
+func (c *AzureCredentialsCreatePayload) SetOrganizationId(organizationId string) {
+	c.OrganizationId = organizationId
+}
```

Comment:
I don't know if it is related. but maybe after you start to support [genric](https://github.com/env0/terraform-provider-env0/pull/417#pullrequestreview-1003569256) you can bundle it

Reply:
Generics is still a little limited in 1.18.
I'll see if I can work around it. Otherwise, may need to revisit in go 1.19.

Reply:
@HeverFarber 

I tried, but it seems like a limitation in golang 1.18 generics (I cannot access struct fields).
More details in https://stackoverflow.com/questions/70358216/how-can-i-define-a-struct-field-in-my-interface-as-a-type-constraint-type-t-has

We could revisit it in the future (if the remove the restriction in go1.19).

Generics is very recent... so it has some limitations at the moment /: